edxval/config/waffle.py

```diff
@@ -8,21 +8,7 @@
 
 WAFFLE_NAMESPACE = 'edxval'
 
-
-def waffle_name(toggle_name):
-    """
-    Method to append waffle namespace to toggle's name
-
-    Reason behind not using f-strings is backwards compatibility
-    Since this is a method, it should be easy to change later on
-    """
-    return "{namespace}.{toggle_name}".format(
-        namespace=WAFFLE_NAMESPACE,
-        toggle_name=toggle_name,
-    )
-
-
-# .. toggle_name: OVERRIDE_EXISTING_IMPORTED_TRANSCRIPTS
+# .. toggle_name: edxval.override_existing_imported_transcripts
```

Comment:
@edx/core-committer-regisb: I’ve seen this issue elsewhere. Could we lint for “.” In the name for waffle flags and switches? Related, do we need to add the non namespaced version of these classes to the list of implementations for the annotation, or do we not want to add every class and subclass?

Reply:
> Could we lint for “.” In the name for waffle flags and switches?

Well, we could certainly do it, but then annotations would fail on [NonNamespaced*](https://github.com/edx/edx-platform/blob/master/openedx/core/djangoapps/waffle_utils/docs/decisions/0004-waffle-util-namespacing.rst#decision) objects, so I don't think we should do it.

> do we need to add the non namespaced version of these classes to the list of implementations for the annotation, or do we not want to add every class and subclass?

I think that the class used for the implementation is of no concern to the end user, so we should not be adding it.

Reply:
@regisb: I agree, but still wish there was a solution to help avoid common problems. One idea would be to take the short flag name (i.e. everything after the period, if there even is one), and check to ensure the same appears in the name parameter of the call to the constructor? Just brainstorming for the future. This may also be risky and a bad idea.